app/controllers/notificationController.js

```diff
@@ -27,24 +27,27 @@ app.controller('NotificationController', function ($controller, $scope, Notifica
 
     $scope.forms = {};
 
-    $scope.notificationToDelete = {};
+    var emptyNotification = {
```

Comment:
```suggestion
    const emptyNotification = {
```

I suggest using `const` whenever reasonable and possible.
I don't see `emptyNotification` being re-assigned anywhere.

Reply:
I left it as var for consistency. Refactoring for later ECMA version is scope creep.

https://github.com/search?q=repo%3ATAMULib%2FLibraryServiceStatusSystemUI%20var&type=code
https://github.com/search?q=repo%3ATAMULib%2FLibraryServiceStatusSystemUI+const&type=code

Reply:
There is this one:
- https://github.com/TAMULib/LibraryServiceStatusSystemUI/blob/2809f72219d3f014cd6b543b9cff4718d0563172/app/controllers/notificationController.js#L68


But I am fine with going forward as it is AngularJS (aka old).